convert2rhel/cert.py

```diff
@@ -73,4 +73,8 @@ def remove(self):
             os.remove(self._target_cert_path)
             loggerinst.info("Certificate %s removed" % self._target_cert_path)
         except OSError as err:
-            loggerinst.error("OSError({0}): {1}".format(err.errno, err.strerror))
+            if err.errno == 2:
```

Comment:
I'd for readability for this:
```suggestion
            if err.errno == errno.ENOENT:
```

Needs of course, `errno` to be imported.

Reply:
Thank you for the help with this Vinzenz! I will implement this into the code!